src/generator.cs

```diff
@@ -5191,9 +5182,10 @@ void PrintMethodAttributes (MemberInformation minfo)
 			PrintPlatformAttributes (minfo.method.DeclaringType);
 		}
 
-		// in theory we could check for `minfo.is_ctor` but some manual bindings are using methods for `init*`
-		if (AttributeManager.HasAttribute<DesignatedInitializerAttribute> (mi))
+		foreach (var di in AttributeManager.GetCustomAttributes<DesignatedInitializerAttribute> (mi)) {
```

Comment:
Is this change where you change the check from a single attribute to multiple attributes intentional? IIRC @spouliot did the reverse not long ago.

Reply:
No, it wasn't intentional. It happened because I was copying stuff from an older file. I thought I had fixed it. Should be fixed now.